### PR TITLE
Add bruteforce capabilities

### PR DESCRIPTION
**Pull request by rullzer**
@LukasReschke @tobiasKaminsky as discussed

### CODE REVIEW

File: lib/composer/composer/autoload_classmap.php [hunk 1]
```diff
@@ -572,6 +572,7 @@
     'OC\\Files\\Cache\\Wrapper\\CacheJail' => $baseDir . '/lib/private/Files/Cache/Wrapper/CacheJail.php',
     'OC\\Files\\Cache\\Wrapper\\CachePermissionsMask' => $baseDir . '/lib/private/Files/Cache/Wrapper/CachePermissionsMask.php',
     'OC\\Files\\Cache\\Wrapper\\CacheWrapper' => $baseDir . '/lib/private/Files/Cache/Wrapper/CacheWrapper.php',
+    'OC\\Files\\Cache\\Wrapper\\JailPropagator' => $baseDir . '/lib/private/Files/Cache/Wrapper/JailPropagator.php',
```
Comment (LukasReschke) [inline lib/composer/composer/autoload_classmap.php]: Master tests  broken or why is this mandatory?

Comment (rullzer) [inline lib/composer/composer/autoload_classmap.php]: mmm I guess something is broken then....

Comment (rullzer) [inline lib/composer/composer/autoload_classmap.php]: taken care of in #5703

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by LukasReschke — APPROVED** [review]

**Pull request merged**